src/helper/connector.js

```diff
@@ -0,0 +1,34 @@
+define([], function () {
```

Comment:
I see. We used to use it on our projects. Today we go more for things like Browserify or Webpack which gives us a cleaner code. We only need to require stuff in the Node way and that's it.